reaktive/src/commonTest/kotlin/com/badoo/reaktive/testutils/TestObservableObserverExt.kt

```diff
@@ -0,0 +1,23 @@
+package com.badoo.reaktive.testutils
```

Comment:
Do you want to add `Observable.test()` extension function as it is done in RxJava? It will be much simpler to use it later.